types/react-select-country-list/index.d.ts

```diff
@@ -18,6 +18,10 @@ declare namespace ReactSelectCountries {
         labelMap: LabelValueMap;
         valueMap: LabelValueMap;
     }
+
+    interface NativeCountries extends Countries {
+        nativeData: CountryData[];
```

Comment:
is this a private-like reference? There is no public api doc for this property

Reply:
The docs are indeed not complete. They describe the methods but not the full CountryList object that is returned when calling native(). When calling native(), the nativeData property is added to the CountryList object. This nativeData property had the wrong type associated with it.

See [here](https://github.com/Chun-Lin/react-select-country-list/blob/master/country-list.js#L63) where the nativeData is added to the object.

Reply:
I understand the prop being added and expanded object being returned. That's OK, but I don't see how this could be useful, adding this 'nativeData' just for the sake of adding this. Is tis property really consumed by code using this JS module?

Reply:
I use the nativeData property to fill a country select list with native country names instead of all English, but I see your point: It is also possible to use getValueList or getLabelList. In this case the return type of native() should be "this". Unfortunately the api is a bit wonky here as for example getValues() will still return the English names instead of native.

Reply:
cool, let's go pragmatic. If it works, that's ok